### PR TITLE
solved rotten-tomatoes

### PR DESCRIPTION
**Pull request by engineer-kufre**
Challenge solved. Please review and merge PR

### CODE REVIEW

File: src/rotten-tomatoes/index.js [hunk 1]
```diff
@@ -1,3 +1,58 @@
...
+  // Helper function to check adjacent elements
+  // Turn twos to zeros and ones to twos
+  // Keep track of number of minutes
+  function checkAdjacent(grid, numberOfMinutes) {
+    let queue = [];
+    for (let row = 0; row < grid.length; row++) {
+      for (let col = 0; col < grid[0].length; col++) {
+        if (grid[row][col] === 2) {
+          queue.push([row, col]);
+          grid[row][col] = 0;
```
Comment (bondz) [inline src/rotten-tomatoes/index.js:13]: Why do you need to turn them to `0`?

File: src/rotten-tomatoes/index.js [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+  }
+
+  // If fresh tomato remains untouched
+  for (let row = 0; row < grid.length; row++) {
+    for (let col = 0; col < grid[0].length; col++) {
+      if (grid[row][col] === 1) {
+        return -1;
+      }
+    }
+  }
```
Comment (bondz) [inline src/rotten-tomatoes/index.js:54]: You can use `[].some` to simplify this.

**Review by bondz — APPROVED** [review]
Nice.

File: src/rotten-tomatoes/index.js (hunk 1 above)
Comment (engineer-kufre) [reply src/rotten-tomatoes/index.js:13]: I did that so their indexes are ignored in subsequent iterations.

File: src/rotten-tomatoes/index.js (hunk 2 above)
Comment (engineer-kufre) [reply src/rotten-tomatoes/index.js:54]: Noted. I'll read up on it sir.

**Pull request merged**